packages/webapp/public/locales/es/common.json

```diff
@@ -14,7 +14,7 @@
   "EDIT": "Editar",
   "EDIT_DATE": "Editar fecha",
   "EG": "Ejemplo",
-  "EXPORT": "Exportar",
+  "EXPORT": "MISSING",
```

Comment:
should these translations be removed? same with other translation files

Reply:
it looks like Spanish translations are being removed and Portuguese are being reverted. is this intentional?